src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs

```diff
@@ -10,21 +10,17 @@ public class FoldersViewModel : ObservableObject
 	{
 		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
 
-
-		//FileTag combobox indexes (required to hide SyncStatus)
+		// FileTag combobox indexes (required to hide SyncStatus)
 		private readonly int FileTagSortingIndex = 5;
 		private readonly int FileTagGroupingIndex = 6;
 
-
 		public FoldersViewModel()
 		{
 			SelectedDefaultLayoutModeIndex = (int)DefaultLayoutMode;
 			SelectedDefaultSortingIndex = UserSettingsService.FoldersSettingsService.DefaultSortOption == SortOption.FileTag ? FileTagSortingIndex : (int)UserSettingsService.FoldersSettingsService.DefaultSortOption;
 			SelectedDefaultGroupingIndex = UserSettingsService.FoldersSettingsService.DefaultGroupOption == GroupOption.FileTag ? FileTagGroupingIndex : (int)UserSettingsService.FoldersSettingsService.DefaultGroupOption;
 		}
 
-		// Properties
```

Comment:
These comments help provide seperatation in large view models without the inconvenience of regions

Reply:
Please restore the comments, we don't use regions anymore because they are inconvenient.

Reply:
We're moving away from them and don't want to introduce into new code.

Reply:
Done